src/Infocom.php

```diff
@@ -882,6 +882,41 @@ public static function getAmortTypeName($value)
         }
     }
 
+    public static function getLogServiceName(): string
+    {
+        return 'financial';
+    }
+
+    public static function displayFullPageForItem($id, ?array $menus = null, array $options = []): void
+    {
+        $ic = new self();
+
+        Html::popHeader(self::getTypeName(), $_SERVER['PHP_SELF']);
```

Comment:
This param is not used in the target method.

```suggestion
        Html::popHeader(self::getTypeName());
```